packages/graphql/tests/integration/math.int.test.ts

```diff
@@ -135,7 +135,7 @@ describe("Mathematical operations tests", () => {
 
             const query = /* GraphQL */ `
             mutation($id: ID, $value: ${type}) {
-                ${movie.operations.update}(where: { id_EQ: $id }, update: {viewers_${operation}: $value}) {
+                ${movie.operations.update}(where: { id_EQ: $id }, update: {viewers: {${operation}: $value }}) {
```

Comment:
Unrelated change but it will avoid us to pass again to this test!
```suggestion
                ${movie.operations.update}(where: { id: { equals: $id } }, update: {viewers: {${operation}: $value }}) {
```